ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -171,6 +173,12 @@ module('Integration | Component | auth jwt', function(hooks) {
   });
 
   test('oidc: it calls window.open popup window on login', async function(assert) {
+    // skip this test if running in ie11 because of concurrency problems
+    if (!!window.MSInputMethodContext && !!document.documentMode) {
```

Comment:
What do you think about removing these now that we're not running the whole suite in IE?